designs/administrator/olm/uninstall-operator-delete-operands/index.md

```diff
@@ -0,0 +1,30 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Optional operand delete during operator uninstall
+
+Today when uninstalling an operator, any created operands are not removed with the operator. The uninstall dialog message warns the user that they will need to clean these up manually if they want them removed.
+
+With this design, the list of operands is shown in the uninstall operator dialog, with a checkbox to indicate that these operands should be removed with the operator's uninstall.
+
+## User selects for all operands to be deleted
+
+![Uninstall operator](img/1-1.png)
+- The user begins the operator uninstall process.
+
+![Uninstall operator dialog](img/1-2.png)
+- The uninstall modal now includes a checkbox that allows the user to indicate whether they’d like to also delete the operator’s operands (similar to the ‘delete dependent objects of this resource’ checkbox used on other delete modals in the console.)
+- A list of the operator’s operands are shown for reference.
+- The user can click the operands name to be immediately navigated to that operand’s details view. The user will still be in the context of the operator (just inside the operand) so can fairly easily return to the uninstall process.
+
+![Uninstall operator dialog](img/1-3.png)
```

Comment:
Very small visual nit, but should the checkbox have a check rather than an 'x' in it?

Reply:
Fixed. I should update my sketch libraries older docs! Thanks.